packages/web/src/services/sentry.ts

```diff
@@ -17,7 +17,12 @@ const MAX_BREADCRUMBS = 300
 export const initializeSentry = () => {
   init({
     dsn: env.SENTRY_DSN,
-    ignoreErrors: navigator.userAgent === 'probers' ? [/.*/] : undefined,
+    ignoreErrors:
+      typeof navigator !== 'undefined'
```

Comment:
ahh didn't think about ssr 😨  thank your fixing!!

Reply:
Do worries, its gonna be tough to get used to this